libc/src/stdio/generic/ftello.cpp

```diff
@@ -0,0 +1,26 @@
+//===-- Implementation of ftello ------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/stdio/ftello.h"
+#include "src/__support/File/file.h"
+
+#include "src/errno/libc_errno.h"
+#include <stdio.h>
```

Comment:
Do you need anything from stdio.h? (here and in libc/src/stdio/generic/fseeko.cpp, too)

Reply:
Hm... No not really. I did not think about the includes so much. Got them from the ftell.cpp file.

Reply:
Shall I update the include statements for ftell/fseek also within this PR?

Reply:
sure!

Reply:
Done.